Makefile

```diff
@@ -58,7 +58,7 @@ ifndef $(shell command -v prototool help > /dev/null)
 endif
 	prototool lint
 
-protoc: protodocs
+protoc: #protodocs
```

Comment:
req: please make sure that you not reverting the protodocs change. It is important for our clients.

Reply:
Sorry for that, I always forget to uncomment it :/